cockatrice/src/setsmodel.cpp

```diff
@@ -80,6 +82,9 @@ QVariant SetsModel::headerData(int section, Qt::Orientation orientation, int rol
         case IsKnownCol:
             return QString(
                 "Is known"); /* no tr() for translations needed, column is just used for sorting --> hidden */
+        case PriorityCol:
```

Comment:
if the priority is never shown to users, and the sorting is moved to the setslist, what benefit does adding the priority column to the model have?